sphinx/source/gallery/index.rst

```diff
@@ -0,0 +1,9 @@
+===============
```

Comment:
will be moved to separate task, pending update of the [`Diataxis`](https://diataxis.fr/) adoption plan.